.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     rev: v0.2.0
     hooks:
     -   id: blacken-docs
-        additional_dependencies: [black==18.6b1]
+        additional_dependencies: [black==18.6b2]
```

Comment:
@asottile can we make this additional dependency to be also auto-updated somehow?

Reply:
`pre-commit` currently doesn't know much about the contents of `additional_dependencies`, it's certainly _possible_ though I imagine pretty difficult